### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by allanrenucci**

### CODE REVIEW

File: src/main/scala/scala/xml/quote/quote.scala [hunk 1]
```diff
@@ -6,10 +6,7 @@ import scala.xml.quote.internal.QuoteImpl
 package object quote {
   implicit class XmlQuote(ctx: StringContext) {
     object xml {
-      def apply[T](args: T*): scala.xml.Node = macro QuoteImpl.apply[T]
-    }
-    object xmls {
-      def apply[T](args: T*): scala.xml.NodeBuffer = macro QuoteImpl.applySeq[T]
+      def apply[T](args: T*): AnyRef = macro QuoteImpl.apply[T]
```
Comment (allanrenucci) [inline src/main/scala/scala/xml/quote/quote.scala:9]: @densh The macro expends to either `scala.xml.NodeBuffer` or a subtype of `scala.xml.Node`.
What should I put as a return type? `AnyRef` will drive IDE crazy...

**Comment by allanrenucci** [comment]
I removed `Unliftables.scala`, I don't see its purpose. Am I missing something?

**Comment by densh** [comment]
@allanrenucci It's all good, they are not necessary for xmlquote. 👍

File: src/main/scala/scala/xml/quote/quote.scala (hunk 1 above)
Comment (densh) [reply src/main/scala/scala/xml/quote/quote.scala:9]: I think we have no choice but to return `AnyRef` here, the two types are completely unrelated.

**Comment by allanrenucci** [comment]
Superseded by #16

**Pull request closed without merging**